.gitignore

```diff
@@ -0,0 +1,36 @@
+# Package distribution
+dist
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+
+# Dependency directories
+node_modules
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+.env.test
+
+# parcel-bundler cache (https://parceljs.org/)
```

Comment:
Just curious, where did this `.gitignore` come from?

Reply:
Generated by Github I think.

Reply:
Nope! Was just wondering if this was the GitHub generated one. The norms change so quickly haha